services/web-app/src/components/home/Hero.tsx

```diff
@@ -2,9 +2,10 @@ import { Button, Link } from "@radix-ui/themes";
 import { motion, useScroll, useSpring, useTransform } from "framer-motion";
 import React, { useRef } from "react";
 import { MdNorthEast } from "react-icons/md";
+
```

Comment:
[Note] These changes had been made as linting hadn't been run on the code Aiden added